api/src/middleware/cache.ts

```diff
@@ -42,6 +42,7 @@ const checkCacheMiddleware: RequestHandler = asyncHandler(async (req, res, next)
 
 		res.setHeader('Cache-Control', getCacheControlHeader(req, cacheTTL));
 		res.setHeader('Vary', 'Origin, Cache-Control');
+		res.setHeader('x-directus-cache', 'HIT');
```

Comment:
Lets only add the cache header when the cache is enabled through `CACHE_ENABLED`, and lets add another env var to control the name of the cache cache maybe? Not sure if that's too overkill, but I can imagine people might want to white label that header name 🤔 That flag would also allow for `false` to disable the header

Reply:
Sure! Can add the label env flag. If not set, will not set the header.

As for the cache enabled: if it's not enabled that piece of code won't be reached right?

Reply:
True! I was already thinking another step ahead in that we should also render a "MISS" when the cache is enabled but not used 🙂  In that case, it might also be hit when the cache is not used